src/main.py

```diff
@@ -22,19 +22,19 @@
 valid_path = '../mini_imagenet/csvsplits/valid.csv'
 test_path = '../mini_imagenet/csvsplits/test.csv'
 separator = ';'
-model_best_state_filename = 'best_state.pt'
+filename = 'best_protonet.pt'
```

Comment:
filename c'est plus cute, mais moins explicite (moins claire). Voici ce que Clean Code dit sur les noms de variables : "The name of a variable, function, or class, should answer all the big questions. It should tell you why it exists, what it does, and how it is used."